### DISCUSSION TITLE
init ca in docker

### DISCUSSION THREAD

**Discussion by JiangP95** [smallstep/certificates#842] [Q&A]
What command was executed  when ”docker run smallstep/step-ca“  ， I want to execute this command manually。

**Comment by maraino** [comment]
Hi @JiangP95, you can see it in the [Dockerfile](https://github.com/smallstep/certificates/blob/master/docker/Dockerfile.step-ca). The [entrypoint.sh](https://github.com/smallstep/certificates/blob/master/docker/entrypoint.sh) there is used to create an initial configuration if it doesn't exist and some environment variables are passed.
